src/main/java/edu/hawaii/its/api/groupings/MemberAttributeResults.java

```diff
@@ -0,0 +1,58 @@
+package edu.hawaii.its.api.groupings;
+
+import edu.hawaii.its.api.wrapper.Subject;
+import edu.hawaii.its.api.wrapper.SubjectsResults;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MemberAttributeResults implements MemberResults<MemberResult> {
+
+    private String resultCode;
+    private List<String> invalid;
+    private List<MemberResult> results;
+
+    public MemberAttributeResults() {
+        setResultCode("FAILURE");
+        setResults(new ArrayList<>());
+    }
+
+    public MemberAttributeResults(List<String> invalid) {
+        setResultCode("FAILURE");
+        setInvalid(invalid);
+    }
+
+    public MemberAttributeResults(SubjectsResults subjectsResults) {
+        setResultCode(subjectsResults.getResultCode());
+        setResults(subjectsResults.getSubjects());
+    }
```

Comment:
Make sure to `setResults(new ArrayList<>())` and `setInvalid(new ArrayList<>())` for all these constructors, we do not want any null values in the result.